live-usb-creator/install_scripts/0_post_install_nochroot

```diff
@@ -58,12 +58,20 @@ cp /vagrant/syslinux-splash.png /mnt/sysimage/usr/share/anaconda/boot/syslinux-s
 ###############################################################################
 
 mkdir -p /mnt/sysimage/data/app/subzero
-cp -r /vagrant/data_app_subzero/* /mnt/sysimage/data/app/subzero
```

Comment:
Thanks for getting rid of this. Now the code is easier to read.

Reply:
Yeah, the file globbing could have been more explicit.